packages/report-server/src/__tests__/reportBuilder/transform/parser/functions.test.ts

```diff
@@ -75,6 +75,27 @@ describe('functions', () => {
     });
   });
 
+  describe('context', () => {
+    describe('orgUnitCodeToName', () => {
+      const context = {
+        orgUnits: [
```

Comment:
Totally happy with `orgUnits` for the name here.

At the same time it makes me sad... I feel we're losing the war on the 'Rename orgUnits to entities' fight. Org units is the right choice here, since the analytic has 'organisationUnit' in it, but the terminology is just so entrenched 😢

Reply:
Also, wanna note that I love how the architecture allows us to write tests like this! Super clean 👍

Reply:
Yeah I think we came up with something clean! 🙌